### PR TITLE
Add nvme-cli package.

### PR DESCRIPTION
**Pull request by L38s**
# Purpose of pull request

nvme-cli 

This pr adds nvme-cli package based on meta-openembedded's meta-oe/recipes-bsp/nvme-cli/nvme-cli_1.6.bb

# Test
## How to test

Check the file structure and commands.

## Test result

Configure the file.
```
root@qemuarm64:~# find / -name "*nvme*"
/usr/share/zsh/site-functions/_nvme
/usr/sbin/nvme
/sys/bus/nvmem
```
Check the commands.
```
root@qemuarm64:~# nvme list
```
Since there was no device, I specified the name of the device listed on the web.
```
root@qemuarm64:~# nvme format /dev/nvme0 -s 2 -n 1
/dev/nvme0: No such file or directory
```

### CODE REVIEW

**Comment by masami256** [comment]
>  This pr adds FOOBAR package based on meta-openembedded's meta-oe/recipes-bsp/nvme-cli/nvme-cli_1.6.bb

@L38s please correct this information.

File: recipes-debian/nvme-cli/nvme-cli_debian.bb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#PV .= "+git${SRCPV}"
+
+inherit debian-package
+require recipes-debian/sources/nvme-cli.inc
+
+FILES_${PN} += "\
+    ${datadir}/zsh \
+"
+
+SRCREV = "642d426faf8a67ed01e90f7c35c0d967f8cc52a3"
```
Comment (masami256) [inline recipes-debian/nvme-cli/nvme-cli_debian.bb]: Does this recipe use SRCREV?

File: recipes-debian/nvme-cli/nvme-cli_debian.bb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# base commit: a24acf94d48d635eca668ea34598c6e5c857e3f8
+
+SUMMARY = "NVMe management command line interface"
+AUTHOR = "Stefan Wiehler <stefan.wiehler@missinglinkelectronics.com>"
+HOMEPAGE = "https://github.com/linux-nvme/nvme-cli"
+SECTION = "console/utils"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=8264535c0c4e9c6c335635c4026a8022"
+DEPENDS = "util-linux"
+#PV .= "+git${SRCPV}"
```
Comment (masami256) [inline recipes-debian/nvme-cli/nvme-cli_debian.bb]: you can remove it.

**Comment by L38s** [comment]
Fixed.

**Comment by L38s** [comment]
Sorry, I thought I fixed everything, but I didn't.
I fixed two places again.

**Comment by masami256** [comment]
@L38s thanks! I'll merge this pr when ci is passed.

**Pull request merged**